src/ImageSequencer.js

```diff
@@ -237,6 +237,32 @@ ImageSequencer = function ImageSequencer(options) {
     return `sequencer -i [PATH] -s ${cliStringSteps} -d '${JSON.stringify(cliOptions)}'`
   }
 
+  // Checks if input is a string of comma separated module names
+  function detectSyntax(str) {
+    let result = (str.includes(',') || str.includes('{')) ? true : false
```

Comment:
The ternary operator `?` is not required as it just returns true/false

Reply:
You can directly return it as well.
```js
return (str.includes(',') || str.includes('{'))
```

Reply:
Won't make much of a difference, i guess?

Reply:
Yeah. Just makes the code cleaner. If you are going to push more commits, change this as a side effect.